<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="playButton">
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="topRow">
      <item>
       <widget class="QPushButton" name="insertStopPointButton">
        <property name="text">
         <string>Insert stop point</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="insertStartPointButton">
        <property name="text">
         <string>Insert start point</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="positionWidget" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="timeAndList">
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="text">
         <string>0.00 / 0.00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="syncButton">
      <property name="text">
       <string>Sync</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="synchroPointList"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionSave_as"/>
    <addaction name="actionOpen"/>
    <addaction name="actionImport_Audio"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuSync_points">
    <property name="title">
     <string>Sync points</string>
    </property>
    <addaction name="actionInsert_stop_point"/>
    <addaction name="actionInsert_start_point"/>
    <addaction name="actionSync"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuSync_points"/>
  </widget>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open project</string>
   </property>
  </action>
  <action name="actionImport_Audio">
   <property name="text">
    <string>Import Audio</string>
   </property>
  </action>
  <action name="actionInsert_stop_point">
   <property name="text">
    <string>Insert stop point</string>
   </property>
  </action>
  <action name="actionInsert_start_point">
   <property name="text">
    <string>Insert start point</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="text">
    <string>Sync</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
